gcp_variant_transforms/libs/beam_util.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
Should this file be named `counters_util` or `counter_factory`(preferred)  instead? In general, we should aim to limit the scope of 'util' classes as they can become a bag of unrelated stuff over time.